server.go

```diff
@@ -1723,14 +1723,35 @@ const bootstrapBackOffCeiling = time.Minute * 5
 func (s *server) initialPeerBootstrap(ignore map[autopilot.NodeID]struct{},
 	numTargetPeers uint32, bootstrappers []discovery.NetworkPeerBootstrapper) {
 
+	// We'll start off by waiting 2 seconds between failed attempts, then
+	// double each time we fail until we hit the bootstrapBackOffCeiling.
+	delayTime := time.Second * 2
+	delayTicker := time.NewTicker(delayTime)
+	defer delayTicker.Stop()
```

Comment:
Since we're re-initializing this ticker below, are we sure that this `Stop` won't apply to the first initialization?

Reply:
You mean it'll still be pointing to the original ticker? We use this pattern a few places within the codebase, FWIW.

Reply:
Switched to manually manage a `time.After` instance.